develop/api_guc.rst

```diff
@@ -277,6 +277,17 @@ Specifies whether to automatically propagate DDL changes from the coordinator to
 
   For a list of DDL propagation support, see :ref:`ddl_prop_support`.
 
+.. _enable_local_ref_fkeys:
+
+citus.enable_local_reference_table_foreign_keys (boolean)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+Enables foreign keys between reference tables and local tables when coordinator is added to metadata (see :ref:`citus_add_node`).
+When enabled, Citus supports foreign keys between reference tables and local tables by applying a conversion to local tables (i.e adds local tables to Citus metadata) when defining such foreign keys.
```

Comment:
looked into deployment, seems that I should add some new lines like we did in `citus.all_modifications_commutative`

Reply:
Do we have any docs listing the limitations of such foreign keys (e.g., ON DELETE CASCADE not supported in some cases)?

Reply:
I think I should mention such limitations in http://docs.citusdata.com/en/v9.5/develop/reference_ddl.html?highlight=Indices#adding-removing-constraints

Reply:
and give a reference to this section